front/src/applications/stdcmV2/hooks/useHorizontalScroll.ts

```diff
@@ -0,0 +1,111 @@
+import { useRef, useState, useEffect, useMemo } from 'react';
+
+import { debounce } from 'lodash';
+
+const DEBOUNCE = 100;
+const useHorizontalScroll = (itemSelectorClassname: string, itemToShowCount: number) => {
+  const scrollableRef = useRef<HTMLDivElement>(null);
+  const [showLeftBtn, setShowLeftBtn] = useState(false);
+  const [showRightBtn, setShowRightBtn] = useState(false);
+  const selector = `.${itemSelectorClassname}`;
+
+  // Calculate the width of an item including its margin-right
```

Comment:
why not the margin left ? (maybe we can update the comment)

Reply:
Yes, For more robustness we should add margin left too ;)

Reply:
Done